bin/process-manifests.sh

```diff
@@ -215,24 +215,30 @@ function process_manifests() {
 
 
 >&2 echo "On branch $GIT_BRANCH"
-manifests=$( find $GEARS_DIR $BOUTIQUES_DIR -iname "*.json" )
+if [ -z "$GIT_COMMIT_SENTINEL" ]; then
+    >&2 echo "Using all manifests"
+    manifests=$( find $GEARS_DIR $BOUTIQUES_DIR -iname "*.json" )
+    >&2 echo "$manifests"
+else
+    >&2 echo "Using updated manifests"
+    manifests=$( git diff --name-only $GIT_COMMIT_SENTINEL | grep -e "^$GEARS_DIR/..*$" -e "^$BOUTIQUES_DIR/..*$" || true)
```

Comment:
Basically. I didn't see that the find above is limiting to the boutique and gear folders.

We can leave this grep as it is, or we can change it around to clarify. I'm fine with either.